torch/utils/_foreach_utils.py

```diff
@@ -5,7 +5,34 @@
 from torch import Tensor
 from torch.autograd.grad_mode import no_grad
 
+__all__ = ["SupportForeachDeviceType"]
 
+class SupportForeachDeviceType(object):
+    r"""
+    A class that manages the default device type for foreach support,
+    default is 'cuda'.
+    """
+    _default_device_type = "cuda"
```

Comment:
```suggestion
    _default_devices = ["cuda"]
```
Why not have a list of supported devices? Will there never be the case where there may be tensors on multiple devices (across CUDA and the custom device type)? Even if there isn't, why is it better to specify only 1 supported device over a general list of supported devices?